vtex-apps/admin-pages
# feat: new translations added for enableCustomCurrencySymbol

#### What problem is this solving?

Currently, the user is not able to get the value from the custom currency symbol in Trade Policy, this feature allows the user to retrieve it and now set it on the UI, and now, with translations for different languages 😬

#### How to test it?

Go to [admin/cms/store](https://iespinoza2--koestore.myvtex.com/admin/cms/store) and check translations for enableCustomCurrencySymbol

<img width="452" alt="image" src="https://user-images.githubusercontent.com/13649073/178060504-4d199698-5c47-4231-9404-67643a011981.png">

#### Related to / Depends on

This other PR with the feature: https://github.com/vtex-apps/format-currency/pull/29

This other PR adding the flag in the UI: https://github.com/vtex-apps/store/pull/556

Zendesk 590112

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/3orif0Pxk3I4WQj46k/giphy.gif)
